src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline

```diff
@@ -4969,6 +4969,312 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "yard", "~> 0.9"
 end
 
+============== file: samples/babble_about_book_request_streaming_client_empty_response_type_with_response_handling.rb ==============
+//FIXME: license section
+
+# DO NOT EDIT! This is a generated sample ("RequestStreamingClient",  "empty_response_type_with_response_handling")
+# FIXME: import section
+# FIXME: sample function declaration
+# FIXME: initialize client
+
+request = {}
+
+requests = [request]
+library_service_client.babble_about_book(requests)
+# FIXME: response handling
+# FIXME: main function
+============== file: samples/babble_about_book_request_streaming_client_empty_response_type_without_response_handling.rb ==============
+//FIXME: license section
+
+# DO NOT EDIT! This is a generated sample ("RequestStreamingClient",  "empty_response_type_without_response_handling")
+# FIXME: import section
+# FIXME: sample function declaration
+# FIXME: initialize client
+
+request = {}
+
+requests = [request]
+library_service_client.babble_about_book(requests)
+# FIXME: response handling
+# FIXME: main function
+============== file: samples/delete_shelf_request_empty_response_type_with_response_handling.rb ==============
+//FIXME: license section
+
+# DO NOT EDIT! This is a generated sample ("Request",  "empty_response_type_with_response_handling")
+# FIXME: import section
+# FIXME: sample function declaration
+# FIXME: initialize client
+
+formatted_name = Library::V1::LibraryServiceClient.shelf_path("[SHELF_ID]")
+
+library_service_client.delete_shelf(formatted_name)
+# FIXME: response handling
+# FIXME: main function
+============== file: samples/delete_shelf_request_empty_response_type_without_response_handling.rb ==============
+//FIXME: license section
+
+# DO NOT EDIT! This is a generated sample ("Request",  "empty_response_type_without_response_handling")
+# FIXME: import section
+# FIXME: sample function declaration
+# FIXME: initialize client
+
+formatted_name = Library::V1::LibraryServiceClient.shelf_path("[SHELF_ID]")
+
+library_service_client.delete_shelf(formatted_name)
+# FIXME: response handling
+# FIXME: main function
+============== file: samples/discuss_book_request_streaming_bidi_prog.rb ==============
+//FIXME: license section
+
+# DO NOT EDIT! This is a generated sample ("RequestStreamingBidi",  "prog")
+# FIXME: import section
+# FIXME: sample function declaration
+# FIXME: initialize client
+
+name = "BASIC"
+file_name = "comment_file"
+comment = File.binread file_name
```

Comment:
Hehe, I love that... while 90%+ of the Ruby code isn't present... the reading local file feature is present and working.

Thanks for the ping, keep me in the loop!